app/assets/stylesheets/tylium/layout.scss

```diff
@@ -29,6 +29,10 @@
   .breadcrumb-item {
     white-space: nowrap;
 
+    &.active {
+      color: darken($mutedText, 4.15%);
```

Comment:
In the future is there documentation for us to know how much some of these elements will need to be darkened? 4.15%, 10%, 2%. They seem random. I think we need a strategy for this.

Reply:
This is to adjust the output color of the variable to meet the WCAG 2.0 Level AA standards. In terms of how I came up with the value, it was a combination of using WAVE (accessibility evaluation tool), judgment, and trial & error.

2% was for a color that was almost meeting the standard.
10% was for a color that was further off and needed that much of a bump.
4.15% was for a color where a 4% dump was not enough but 4.5 or even 4.25% didn't look good so I started to try smaller increments where the color would be as close as possible to the original color but met the contrast standards.

Reply:
> but met the contrast standards

So is there a doc we can follow on how to assign a color if you were on vacation?

Reply:
I'll type something up for this.